src/mind.rs

```diff
@@ -134,6 +135,9 @@ pub fn α_β_negamax_search(world: WorldState,
         if experienced_α >= β {
             break;
         }
+        if (20000.0 - experienced_α.abs()).abs() < 30.0 {
```

Comment:
30 might not actually be a good number here. i forgot to think about it again, but it should probably something like the sum of all piece values. or just 1000, since *actual* scores are never going to go above like 30 or 50 or whatever.